packages/dictd/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=http://sourceforge.net/projects/dict/
+TERMUX_PKG_DESCRIPTION="Online dictionary client and server"
+TERMUX_PKG_VERSION=1.12.1
+TERMUX_PKG_SRCURL=https://downloads.sourceforge.net/project/dict/dictd/dictd-${TERMUX_PKG_VERSION}/dictd-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=a237f6ecdc854ab10de5145ed42eaa2d9b6d51ffdc495f7daee59b05cc363656
+TERMUX_PKG_DEPENDS="libmaa"
+TERMUX_PKG_BUILD_DEPENDS="libmaa-dev"
+
+termux_step_pre_configure() {
+	LDLAGS+=" -lmaa"
```

Comment:
is this being used? i know you meant `LDFLAGS`, but could this entire step be deleted?

Reply:
Ok, I will try to delete these steps.
Thanks!